sources/api/ecs-settings-applier/src/ecs.rs

```diff
@@ -64,6 +71,7 @@ pub(crate) async fn main() -> () {
 
 async fn run() -> Result<()> {
     let args = parse_args(env::args());
+    SimpleLogger::init(LevelFilter::Info, LogConfig::default()).context(error::LoggerSnafu)?;
```

Comment:
I guess logging was never working prior to this change?

nit: Can we move these changes to a separate commit?

Reply:
That's correct, nothing was logged out. I can put this in a different commit